sdk/eventhub/event-hubs/package.json

```diff
@@ -50,7 +50,7 @@
     "build:samples": "echo Obsolete.",
     "build:test:browser": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
     "build:test:node": "tsc -p . && dev-tool run bundle",
-    "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1 && npm run generate-certs",
+    "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1 && npm run generate-certs && copyfiles -f ./test/internal/node/partitionKeyHashMap.json ./dist-esm/test/internal/node",
```

Comment:
Why did this not break before then?

Reply:
I added this particular test fairly recently and the tests were broken during that period.